lib/datadog.ts

```diff
@@ -8,28 +8,45 @@
 
 import * as cdk from "@aws-cdk/core";
 import * as lambda from "@aws-cdk/aws-lambda";
-import { applyLayers, redirectHandlers, addForwarder } from "./index";
+import { applyLayers, redirectHandlers, addForwarder, applyEnvVariables, defaultEnvVar } from "./index";
+import { Transport } from "./transport";
 
 export interface DatadogProps {
   pythonLayerVersion?: number;
   nodeLayerVersion?: number;
   addLayers?: boolean;
   forwarderARN?: string;
+  flushMetricsToLogs?: boolean;
+  site?: string;
+  apiKey?: string;
+  apiKMSKey?: string;
+  enableDDTracing?: boolean;
+  injectLogContext?: boolean;
 }
 
 export class Datadog extends cdk.Construct {
-  /** allows accessing the counter function */
   scope: cdk.Construct;
   props: DatadogProps;
+  xport: Transport;
```

Comment:
Unless this is a CDK convention, I think we should rename this transport. xport made me think it was a typo of export.

Reply:
Agreed. I've changed it to `transport: Transport`